### ISSUE TITLE
Issue in create Inbox behaviour 

### ISSUE THREAD

**Issue by brijeshs-atharvasystem** [Files-com/files-sdk-ruby#9]
Hi 
I am trying to create a inbox behaviour but it returns me below error

```
Files::APIError: Value is invalid
from /home/ankit/.rvm/gems/ruby-2.5.3@cview/gems/files.com-1.0.31/lib/files.com/api_client.rb:224:in `handle_error_response'
Caused by Faraday::UnprocessableEntityError: the server responded with status 422
from /home/ankit/.rvm/gems/ruby-2.5.3@cview/gems/faraday-1.0.1/lib/faraday/response/raise_error.rb:30:in `on_complete
```
Request code

```
json = { "key": "sample-test",  "title": "Upload here",   "show_on_login_page": true,  "require_registration": false,  "password": "password" }

Files::Behavior.create(value: json.to_s, path: "path", behavior: "inbox")
```
I am getting successful response when I tried with curl request. 

```
curl https://app.files.com/api/rest/v1/behaviors.json \
  -X POST \
  -H 'Content-Type: application/json' \
  -d  '{"path":"PATH","behavior":"inbox","value":{"key":"sample-test","title":"Upload here","require_registration":false,"show_on_login_page":false,"dont_separate_submissions_by_folder":true, "password": "PASSWORD"}}' -H 'X-FilesAPI-Key: API_KEY'
```
Can you please let me know how to pass json value in `value` key?

**Comment by zeisler** [comment]
Looks like the API key is not being include, try the following.

```ruby
Files::Behavior.create({ value: json.to_s, path: "path", behavior: "inbox" }, { api_key: API_KEY })
```

**Comment by brijeshs-atharvasystem** [comment]
@zeisler Thanks for reply. API key is assigned in following method.
`Files.api_key = Settings.ftp_key`

I am getting an error as `Value is invalid` so it is not related to authentication.

**Comment by zeisler** [comment]
@brijeshs-atharvasystem Sorry for the misunderstanding. The correct thing to do is to set the key value to a Ruby Hash, not a string. Unfortunately the SDK does not allow this. A fix will be coming.

**Comment by zeisler** [comment]
This issue is fixed as of [1.0.62](https://rubygems.org/gems/files.com/versions/1.0.62)
